### ISSUE TITLE
Does not work on full framework projects

### ISSUE THREAD

**Issue by captainjono** [tonerdo/coverlet#759]
In practice, it seems like coverlet is only compatible with analysing .net core projects? Could someone please confirm the current compatibility of Coverlet with respect to this bug https://github.com/microsoft/vstest/issues/2278

And update the project start page to reflect this would be great. I am trying to implement this on a .net full project, by calling vstest.console.exe directly with
>$vstestconsolepath = "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\Extensions\TestPlatform\vstest.console.exe"

> $vsTestArguments = @(
>                         $testDlls #.net 4.6.2
>                         "/logger:trx" 
>                         "/InIsolation"
>                         "/collect:`"XPlat Code Coverage`""
>                         "/TestCaseFilter:" + $testCaseFilters
>                         "/testadapterpath:" + (get-location).Path + "\packages\coverlet.collector.1.2.0\build\netstandard1.0\"
>                         )

```
Microsoft (R) Test Execution Command Line Tool Version 16.4.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...

A total of 2 test files matched the specified pattern.
Data collection : Data collector 'XPlat Code Coverage' threw an exception during type loading, construction, or initialization: System.FormatException: Input string was not in a correct format.
   at System.Text.StringBuilder.FormatError()
   at System.Text.StringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.Format(IFormatProvider provider, String format, Object[] args)
   at Microsoft.VisualStudio.TestPlatform.Common.DataCollector.DataCollectionManager.LoadAndInitialize(DataCollectorSettings dataCollectorSettings, String settingsXml).
Data collection : Data collector 'XPlat Code Coverage' threw an exception during type loading, construction, or initialization: System.FormatException: Input string was not in a correct format.
```

**Comment by NPadrutt** [comment]
I have the same issue on TeamCity.

**Comment by MarcoRossignoli** [comment]
Yes the collectors version of coverlet at the moment is supported only for .NET Core app and not for .NET Framework, I'm waiting some answer from vstest team to understand if move to netstandard is ok or no(I think it's possible but I need confirmation), because collector was written by vstest team not by us.

**Comment by MarcoRossignoli** [comment]
dup of https://github.com/tonerdo/coverlet/issues/705

I'll update documentation asap.

**Comment by MarcoRossignoli** [comment]
@NPadrutt @captainjono for now the unique way to use coverlet for .NET Framework is https://github.com/tonerdo/coverlet#msbuild-integration-suffers-of-possible-known-issue and https://github.com/tonerdo/coverlet#net-global-tool-guide-suffers-of-possible-known-issue that suffer some known issue, but not all users hit the problem.

**Comment by NPadrutt** [comment]
@MarcoRossignoli  Thank you! I found a blog post where he integrated it into teamcity via msbuild, so exactly what I was looking for and that worked. 
https://medium.com/@justingoldberg_2282/setting-up-code-coverage-with-net-xunit-and-teamcity-for-a-solution-with-multiple-test-projects-5d0986db788b

**Comment by MarcoRossignoli** [comment]
Glad to hear...and sorry, I think we will support .NET Framework...but I cannot update code without green light from vstest team so I'm blocked for now on it.

**Comment by MarcoRossignoli** [comment]
TL;DR;

The issue is that we cannot load netcoreapp dll into .net framwork process because resolution fails, so the solution should be compile to netstandard so .net framework host(test host in this case) will load coverlet with no problem.

**Comment by NPadrutt** [comment]
thanks @MarcoRossignoli. Your effort is greatly appreciated! :)
